### PR TITLE
refactor: refactor Center list fragment to compose

### LINKED ISSUE #2080: refactor: migrate center list screen to jetpack compose
**Summary:** 

Transitioning from XML to Jetpack Compose offers advantages such as adopting a declarative UI approach, ensuring type-safe UI development, and minimizing boilerplate code.

**Select Type:**

- [ ] UI Improvements
- [ ] UI Add On
- [ ] Improve Existing Code
- [x] New Feature
- [ ] Other

### PR DESCRIPTION
**Pull request by Aditya-gupta99**
Fixes #2080 
Jira [MIFOSAC-169](https://mifosforge.jira.com/browse/MIFOSAC-169)

<center>
    <table>
        <tr>
            <td align="center" valign="center">
<br><center>XML Screen</center>
<video src="https://github.com/openMF/android-client/assets/94394661/584f0c18-5f28-4a38-801d-bd6d3d315009"> 
            </td>
            <td align="center" valign="center">
<br><center>Jetpack Compose</center>
<video src="https://github.com/openMF/android-client/assets/94394661/99b16a56-28ba-4c38-a96b-68638fa1a030">                 
            </td>
        </tr>
    </table>
</center>

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

[MIFOSAC-169]: https://mifosforge.jira.com/browse/MIFOSAC-169?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: feature/collection-sheet/src/main/java/com/mifos/feature/individual_collection_sheet/individual_collection_sheet/ui/IndividualCollectionSheetScreen.kt [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        Column(
+            modifier = Modifier.padding(paddingValues)
+        ) {
+            MifosTabRow(tabContents = tabContents, pagerState = pagerState)
+        }
+    }
+}
+
+enum class IndividualCollectionSheetScreenContents {
+    NEW,
```
Comment (AdityaKumdale) [inline feature/collection-sheet/src/main/java/com/mifos/feature/individual_collection_sheet/individual_collection_sheet/ui/IndividualCollectionSheetScreen.kt:102]: could have more meaningful names, NEW_COLLECTION_SHEET and SAVED_COLLECTION_SHEET

File: feature/center/src/main/java/com/mifos/feature/center/center_list/ui/CenterListScreen.kt [hunk 2]
```diff
@@ -0,0 +1,578 @@
...
+        },
+        snackbarHostState = snackbarHostState,
+        bottomBar = { },
+        floatingActionButton = {
+            FloatingActionButton(
+                onClick = { createNewCenter() },
+                containerColor = BlueSecondary
+            ) {
+                Icon(
+                    imageVector = Icons.Filled.Add,
```
Comment (AdityaKumdale) [inline feature/center/src/main/java/com/mifos/feature/center/center_list/ui/CenterListScreen.kt:156]: Do we have common file for all the icons

File: feature/collection-sheet/src/main/java/com/mifos/feature/individual_collection_sheet/individual_collection_sheet/ui/IndividualCollectionSheetScreen.kt [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+                        ),
+                        color = Black,
+                        textAlign = TextAlign.Start
+                    )
+
+                }
+            )
+        },
+        snackbarHostState = snackbarHostState,
+        bottomBar = { }
```
Comment (AdityaKumdale) [inline feature/collection-sheet/src/main/java/com/mifos/feature/individual_collection_sheet/individual_collection_sheet/ui/IndividualCollectionSheetScreen.kt:91]: I don't think we need bottomBar , code can be removed from core implentation MifosScaffold also

**Review by AdityaKumdale — CHANGES_REQUESTED** [review]

File: feature/collection-sheet/src/main/java/com/mifos/feature/individual_collection_sheet/individual_collection_sheet/ui/IndividualCollectionSheetScreen.kt (hunk 1 above)
Comment (Aditya-gupta99) [reply feature/collection-sheet/src/main/java/com/mifos/feature/individual_collection_sheet/individual_collection_sheet/ui/IndividualCollectionSheetScreen.kt:102]: I used this Enum class to set the name of the table

File: feature/center/src/main/java/com/mifos/feature/center/center_list/ui/CenterListScreen.kt (hunk 2 above)
Comment (Aditya-gupta99) [reply feature/center/src/main/java/com/mifos/feature/center/center_list/ui/CenterListScreen.kt:156]: I used material3 icons, thus no need to add to the common file

File: feature/collection-sheet/src/main/java/com/mifos/feature/individual_collection_sheet/individual_collection_sheet/ui/IndividualCollectionSheetScreen.kt (hunk 3 above)
Comment (Aditya-gupta99) [reply feature/collection-sheet/src/main/java/com/mifos/feature/individual_collection_sheet/individual_collection_sheet/ui/IndividualCollectionSheetScreen.kt]: I have removed

**Pull request merged**